Assets/Scripts/Models/Tile.cs

```diff
@@ -112,7 +112,7 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovmentModifier { get; set; }
+    public float MovementModifier { get; set; }
```

Comment:
@TomMalbran The reason it's public is so that it is accesable from lua. But making it a public variable is easy. Will do when I have time.